### PR TITLE
Toolbar and Headline of the Mobile Version Of My Portfolio

### PR DESCRIPTION
**Pull request by spydaspider**
**Toolbar and Headline of My Portfolio Mobile.**
Locate the index.html in the project folder and view the site.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+.headline-container .image-container .rect-6 {
+  position: absolute;
+  width: 51.99px;
+  height: 9.3px;
+  left: -34px;
+  top: 141.49px;
+  background: #ebebff;
+  transform: rotate(-46.14deg);
+}
+
```
Comment (phelian23) [inline style.css:51]: - [ ] Kindly make sure that your design (Font family, margins, decorations) looks close as possible to that of the Figma design.
- [ ] Also Kindly remove the horizontal scroll on the mobile version

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @spydaspider, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct use of Github Flow
✅ Good commit messages
✅ Good README template

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (spydaspider) [reply style.css:51]: Thank you, Sir! I will attend to it at once.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  <body>
+    <div class="container">
+      <nav class="flex-div">
+        <div class="toolbar-left">
+          <a href="" class="logo"> My Logo </a>
+        </div>
+        <div class="toolbar-right">
+          <a href=""><img src="images/menu-icon.png" alt = "image"/></a>
+        </div>
+      </nav>
```
[the remark this answers is not in the document]
Comment (spydaspider) [reply index.html:19]: Hi Sir! Please I don't understand the second instruction.(Wrap the hamburger icon in the <nav> tag.

[the remark this answers is not in the document]
Comment (phelian23) [reply index.html:19]: @spydaspider, the menu icon (hamburger) should be wrapped in the `<nav>` tag

**Comment by spydaspider** [comment]
Okay I get it now. Thank you.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @spydaspider,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️
- [ ] Kindly add node_modules to .gitignore

 _Check the comments under the review for more changes_.

## Optional suggestions
- [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by spydaspider** [comment]
Will do, thanks!

**Comment by spydaspider** [comment]
**Toolbar and Headline of My Portfolio Mobile.**

Added node_modules to the .gitignore file.
Changed my font into a more beautiful one.

**Review by KayLemba — APPROVED** [review]
Hi @spydaspider   ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

To Highlight 💯
No linters errors ✔️
Descriptive commit messages ✔️

**Optional suggestions**
Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**